### DISCUSSION TITLE
GHC tools removed from macOS-13

### DISCUSSION THREAD

**Discussion by quark17** [actions/virtual-environments#9063] [Installed software]
Our CI suddenly started failing on macOS-13 a few days ago, and it appears to be because `ghc` (and `ghcup`) were removed: PR #8775, and as indicated in the release notes for [20231218 image update](https://github.com/actions/runner-images/releases/tag/macos-13%2F20231218.2).  Is there any information on why this was done?  If we need GHC for a job, should we update the job to install `ghcup` itself, or do you recommend something else?  Or will it be added back later?

Also, how can I stay informed of changes?  Is watching this repo the only option, or is there some way to watch just the releases, or a mailing list that just has releases and plans?

**Comment by Alexey-Ayupov** [comment]
Hello @quark17. It was decided to remove `ghc`  and `ghcup` from macOS 13 image due to maintenance concerns. Since the macOS 13 image was at that time in the public Beta we didn't create separate announcement for package removal. For all major changes in the GA images we normally create announcements what you can read in the [issues](https://github.com/actions/runner-images/issues?q=is%3Aopen+is%3Aissue+label%3AAnnouncement) section. 

The [runner-images](https://github.com/actions/runner-images) repository is the main source of information regarding changes in the images, please check it regularly.  We are really sorry for making such issue for you, to mitigate it please install `ghc` in runtime.
